app/helpers/application_helper.rb

```diff
@@ -1,2 +1,18 @@
 module ApplicationHelper
+  def bootstrap_class_for flash_type
```

Comment:
I would rename this method 'flash_class' or something a little more clear. That's a very cool feature of Bootstrap that you found!